deploy/gcp/create-service-account.sh

```diff
@@ -7,45 +7,72 @@
 # so it cannot just be added to the existing terraform.
 
 set -euo pipefail
-cd "$(dirname "$0")"
-PROJECT="${TF_VAR_GCP_PROJECT:-$(cat terraform.tfvars | awk -F '=' '/GCP_PROJECT/ {print $2}' | cut -d '"' -f 2)}"
-echo "using project: $PROJECT"
 
-cred_file=credentials.auto.tfvars
-if test -f "$cred_file" ; then
-  if cat "$cred_file" | awk -F'=' '/GCP_CREDENTIALS/ {print $2}' >/dev/null ; then
-    echo "GCP_CREDENTAILS_PATH already set in $cred_file"
+if ! cd "$(dirname "$0")"; then
+    printf "Could not change to base directory of script." >&2
     exit 1
-  fi
 fi
 
-GCLOUD="gcloud --project $PROJECT"
+if ! [[ -d .terraform ]]; then
```

Comment:
The way the docs are written now, someone would run this before running `terraform init`. https://pingcap.com/docs/v3.0/tidb-in-kubernetes/deploy/gcp-gke/

Reply:
This new approach uses `terraform console` which is much more reliable. We can update the code snippet in the docs to have `terraform init` before running this.